src/libraries/System.Net.Sockets/src/System/Net/Sockets/Socket.Unix.cs

```diff
@@ -11,6 +11,25 @@ namespace System.Net.Sockets
 {
     public partial class Socket
     {
+        public Socket(SocketInformation socketInformation)
+        {
+            //
+            // This constructor works in conjunction with DuplicateAndClose, which is not supported on Unix.
+            // See comments in DuplicateAndClose.
+            //
```

Comment:
Do we use this style of commenting elsewhere in System.Net.Sockets, with a blank comment line above and below?  We've generally moved away from that elsewhere.

Reply:
Occasionally yes, it was was used in the methods I changed. Getting rid of it.